### DISCUSSION TITLE
can the explore command show ansi text?

### DISCUSSION THREAD

**Discussion by wangzaixiang** [nushell/nushell#11060] [Q&A]
I having a text which contains ansi colors, when using the explore command, the colors can't show on terminal. 

but using the table command, it is ok.

eg:
let a1 = { name: "a", content:  $"(ansi blue)Hello(ansi reset) world" }

$a1 | table
<img width="271" alt="image" src="https://github.com/nushell/nushell/assets/1256481/d2f7dd71-dbbf-4482-b2da-67e64abf2578">

$a1 | explore
<img width="282" alt="image" src="https://github.com/nushell/nushell/assets/1256481/d7f29197-8a7c-41eb-b30f-e2fa30b03074">

How can i show the ansi-color in the explore mode? (i write a tool, which need to explore a complex SQL statement, by colorful the sql, it looks more clearly)

**Comment by fdncred** [comment]
You can't. `explore` is meant to use the same color theming that nushell uses.

If you want to create color text, you have to do it manually without using `explore` right now.
